### PR TITLE
fix(data): Import Bachittar Natak from SGPC 

### LINKED ISSUE #60: Bachittar Natak
Missing text and subsections

### PR DESCRIPTION
**Pull request by Sarabveer**
### Summary of PR
Initial pass of Bachittar Natak using `Das Granthi. SGPC, Sri Amritsar, Jul 2014.`

### Tests for unexpected behavior
- none_whatsoever

### Time spent on PR
Months

### Linked issues
Fix #60

### Reviewers
@bhajneet @Harjot1Singh

### CODE REVIEW

**Comment by bhajneet** [comment]
Can you please share the source with me somehow?

**Comment by Sarabveer** [comment]
> Can you please share the source with me somehow?

@bhajneet You can use the 2006 edition. https://archive.org/stream/DasGranthi

File: data/Sri Dasam Granth/0043.json [hunk 1]
```diff
@@ -614,7 +614,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "Gt GuMGr GMt surM GmkM ]58]"
+          "SGPC": "Gn GuMGr GMt surM GmkM ]58] "
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0043.json:617]:
```suggestion
          "SGPC": "Gn GuMGr GMt surM GmkM ]58]"
```

File: data/Sri Dasam Granth/0045.json [hunk 2]
```diff
@@ -723,7 +723,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "khw pwiqswhM khw aumrwXM ]"
+          "SGPC": "khw pwqswhM khw aumrwXM ] "
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0045.json:726]:
```suggestion
          "SGPC": "khw pwqswhM khw aumrwXM ]"
```

File: data/Sri Dasam Granth/0053.json [hunk 3]
```diff
@@ -1593,7 +1593,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "iqsY rwj dIXM ]5]"
+          "SGPC": "iqsY rwj dIXM ]5] "
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0053.json:1596]:
```suggestion
          "SGPC": "iqsY rwj dIXM ]5]"
```

File: data/Sri Dasam Granth/0070.json [hunk 4]
```diff
@@ -473,7 +473,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "ieiq sRI bicqR nwtk gMQy husYn bDh ik®pwl ihMmq sMgqIAw bD brnnµ nwm igAwrmo iDAwie smwpqm squ suBm squ ]11]APjU]423]"
+          "SGPC": "ieiq sRI bicqR nwtk gRMQy husYnI bD ik®pwl ihMmq sMgqIAw bD brnnµ nwm igAwrmo iDAwie smwpqm squ suBm squ ]11] APjU ]423] "
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0070.json:476]:
```suggestion
          "SGPC": "ieiq sRI bicqR nwtk gRMQy husYnI bD ik®pwl ihMmq sMgqIAw bD brnnµ nwm igAwrmo iDAwie smwpqm squ suBm squ ]11] APjU ]423]"
```

File: data/Sri Dasam Granth/0039.json [hunk 5]
```diff
@@ -163,53 +188,53 @@
...
-        "id": "VMPD",
+        "id": "F61Z",
         "source_page": 39,
         "source_line": null,
         "pronunciation": null,
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "iqRBMgI CMd ]"
+          "SGPC": "sRI kwl jI kI ausqiq ] "
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0039.json:226]:
```suggestion
          "SGPC": "sRI kwl jI kI ausqiq ]"
```

File: data/Sri Dasam Granth/0050.json [hunk 6]
```diff
@@ -682,7 +682,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "Anµq ssqR JyilXM ]11]"
+          "SGPC": "Anµq ssqR JylXM ]11] "
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0050.json:685]:
```suggestion
          "SGPC": "Anµq ssqR JylXM ]11]"
```

File: data/Sri Dasam Granth/0050.json [hunk 7]
```diff
@@ -1082,7 +1082,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "Ajwq AMg lwhhI ]15]"
+          "SGPC": "Ajwq AMg lwhhI ]15] "
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0050.json:1085]:
```suggestion
          "SGPC": "Ajwq AMg lwhhI ]15]"
```

**Review by bhajneet — COMMENTED** [review]
Excellent work. Looking forward to merging this in soon. Have just a couple questions after reading all the work you did:

1. What's the plan with vishraams? I noticed they were removed in this PR
2. Where do English subsections come from? Same for the English translations at some places

**Comment by Sarabveer** [comment]
>     1. What's the plan with vishraams? I noticed they were removed in this PR

The vishraams were originally from the SearchGurbani database. I have no clue as to what source they were from.

In an earlier discussion we had on this topic, I remember it was decided to keep vishraams per source. But the issue on how to deal with vishraam sources that use SGPC as a base (ex: Talwara Ji) was left open. SGPC itself doesn't publish any vishraams AFAIK.

>     2. Where do English subsections come from? Same for the English translations at some places

The subsection names I did myself. The English Translation for Dasam Granth is from Dr. Surinder Singh Kohli, but there is no PDF and I don't have a physical copy so I cant check if they match the source.

**Comment by bhajneet** [comment]
> I remember it was decided to keep vishraams per source

This doesn't make any sense. Vishraams signal the pauses in gurbani. Based on meanings, poetic meters, etc. If there are gurbani sources drastically differing from others we should understand that first. Otherwise, considering the overall sameness of our gurbani sources, all vishraams should be applicable to all sources.

The question is more how to handle multiple vishraams for each source. Another technical question for @Harjot1Singh

**Comment by bhajneet** [comment]
Given a lack of data, we will tie vishraams directly to the gurmukhi source in which they appear. If in the future, after inputting more data, that it seems easily possible to share vishraams across gurmukhi sources, we will likely create a new major DB version to handle it.

**Review by bhajneet — APPROVED** [review]

**Pull request merged**